package.json

```diff
@@ -54,37 +68,45 @@
     "@storybook/preset-scss": "^1.0.3",
     "@storybook/react": "^7.6.10",
     "@storybook/react-webpack5": "^7.6.10",
-    "@types/jest": "^26.0.13",
-    "@typescript-eslint/eslint-plugin": "^4.21.0",
-    "@typescript-eslint/parser": "^4.21.0",
+    "@types/jest": "^28.1.6",
+    "@types/node": "^16",
+    "@typescript-eslint/eslint-plugin": "^5.43.0",
+    "@typescript-eslint/parser": "^5.43.0",
     "babel-loader": "^8.2.3",
     "css-loader": "5.2.7",
+    "depcheck": "^1.4.3",
     "dependency-tree": "^8.1.2",
-    "eslint": "^7.23.0",
-    "eslint-config-prettier": "^8.1.0",
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-jest": "^24.3.4",
-    "eslint-plugin-jsdoc": "^36.1.0",
+    "eslint": "^8.44.0",
+    "eslint-config-prettier": "^8.8.0",
+    "eslint-plugin-import": "~2.26.0",
+    "eslint-plugin-jest": "^27.2.2",
+    "eslint-plugin-jsdoc": "^39.9.1",
+    "eslint-plugin-n": "^15.7.0",
     "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^3.3.1",
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-promise": "^6.1.1",
     "eslint-plugin-storybook": "^0.6.15",
-    "jest": "^26.4.2",
+    "jest": "^28.1.3",
+    "jest-it-up": "^2.0.2",
     "node-fetch": "^2.6.0",
-    "prettier": "^2.2.1",
-    "prettier-plugin-packagejson": "^2.2.11",
+    "prettier": "^2.7.1",
+    "prettier-plugin-packagejson": "^2.3.0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "rimraf": "^3.0.2",
     "sass": "^1.54.3",
     "sass-loader": "10.1.1",
     "storybook": "^7.6.10",
     "style-loader": "2.0.0",
-    "ts-jest": "^26.3.0",
-    "typescript": "^4.2.4",
+    "ts-jest": "^28.0.7",
+    "ts-node": "^10.7.0",
+    "tsup": "^7.2.0",
+    "typescript": "~4.8.4",
     "webpack": "5"
   },
+  "packageManager": "yarn@3.2.1",
   "engines": {
-    "node": "^16.20.0"
+    "node": "^18.18 || >=20"
```

Comment:
Note that this also bumps the minimum required version of Node from 16 to 18. That's fine but I just wanted to call it out.

Reply:
Great call out! Projects that are currently using the design tokens package and Node v16 will not be compatible with it after this change. So then this would qualify as a breaking change under SemVer guidelines? Therefore, the release incorporating this PR will necessitate a major version increment. 

This should be acceptable given that our three primary consumers of this package have the following Node versions:

- Mobile: "node": "~18.18.2"
- Extension: "node": ">= 18"
- Portfolio: "node": ">=20.0.0"

Reply:
Yes, exactly. This will necessitate a major version increment.